bowling/tests/bowling_tests_gt.cpp

```diff
@@ -0,0 +1,200 @@
+                                                            {{'2', '2'}, {'8', '-'}, {'0', '-'}}};
+    file << "Player1:12|X|5/\n";
+    file << "Player2:3/|54|X\n";
+    file << "Player3:22|8-|0-\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    ASSERT_EQ(game.getPlayers().size(), 3);
+    EXPECT_EQ(game.getPlayers()[0].rolls, expectedAllPlayersRolls[0]);
+    EXPECT_EQ(game.getPlayers()[1].rolls, expectedAllPlayersRolls[1]);
+    EXPECT_EQ(game.getPlayers()[2].rolls, expectedAllPlayersRolls[2]);
+}
+
+TEST_F(GameTests, loadFromFileShouldLoadExtraRolls) {
+    std::vector<Frame> expectedPlayerRolls{{'1', '2'},
+                                           {'3', '4'},
+                                           {'5', '6'},
+                                           {'7', '8'},
+                                           {'9', '9'},
+                                           {'X', ' '},
+                                           {'0', '-'},
+                                           {'5', '/'},
+                                           {'5', '4'},
+                                           {'X', ' '},
+                                           {'4', '4'}};
+    file << "Name:12|34|56|78|99|X|0-|5/|54|X||44\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getPlayers()[0].rolls, expectedPlayerRolls);
+}
+
+TEST_F(GameTests, loadFromFileShouldNotAddEmptyFrameWhenThereAreNoExtraRolls) {
+    std::vector<Frame> expectedPlayerRolls{{'1', '2'},
+                                           {'3', '4'},
+                                           {'5', '6'},
+                                           {'7', '8'},
+                                           {'9', '9'},
+                                           {'X', ' '},
+                                           {'0', '-'},
+                                           {'5', '/'},
+                                           {'5', '4'},
+                                           {'X', ' '}};
+    file << "Name:12|34|56|78|99|X|0-|5/|54|X||\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getPlayers()[0].rolls, expectedPlayerRolls);
+}
+
+std::ostream& operator<<(std::ostream& os, Game::Status status) {
+    os << static_cast<int>(status);
+    return os;
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnNoGameWhenFileIsEmpty) {
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::NO_GAME);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnInProgressWhenFramesAreNotFull) {
+    file << "Robcio:12|27|X\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::IN_PROGRESS);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnFinishedWhenFramesAreFull) {
+    file << "Robcio:12|27|X|1/|22|11|0-|0-|12|0-||\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::FINISHED);
+}
```

Comment:
what about frames with bonus balls? Status is not checked on them. 
Also what when we have many players? What if the first one has completed the game and the last not? What if the first one has bonus balls and the last one not?